### PR TITLE
chore: Add `create_volume` to swagger docs

### PR DESCRIPTION
**Pull request by matheus23**
- Added a `utoipa::path` to the `create_volume` handler
- Updated flake so postgres works in NixOS
- Replaced ucan path dependencies with actual UCAN versions (@blaine this would still cause a runtime error, right? What's the github URL that I could depend on?)

### CODE REVIEW

**Comment by matheus23** [comment]
Hm. We need a way to run kubo during tests.

**Comment by blaine** [comment]
> * Replaced ucan path dependencies with actual UCAN versions (@blaine this would still cause a runtime error, right? What's the github URL that I could depend on?)

Yeah, it's because the `ucan:*` "unscoped" bug in rs-ucan will result in proof chain validation failure. The "fix" I put in the vendored version was here: https://github.com/blaine/rs-ucan/tree/temporary-unscoped-fix (just a one line change), and I'm actually not sure the benefit of having a separate type for unscoped `ucan:` URIs so this _may_ be the best overall fix (plus deleting related code)?

Anyhow, it's just on validating volume changes that things will fail until that issue gets resolved, or we can use a different resource (e.g. `https://{domain}/api/account/{username}/*`) instead of `ucan:*`

File: flake.nix [hunk 1]
```diff
@@ -18,117 +18,114 @@
...
+          PGDATA="./.pg";
+          PGURL=postgres://postgres@localhost:5432/fission-server
+
+          # Initialize a local database if necessary.
+          if [ ! -e $PGDATA ]; then
+            echo -e "\nInitializing PostgreSQL in $PGDATA\n"
+            initdb $PGDATA --no-instructions -A trust -U postgres
+            if pg_ctl -o '-k /tmp' -D $PGDATA start; then
               cd fission-server
-              diesel migration run --database-url $PGURL
```
Comment (blaine) [inline flake.nix:102]: 👍

**Review by blaine — APPROVED** [review]

**Comment by matheus23** [comment]
Ah - sorry for the huge diff in `flake.nix`. I actually have a nix formatter installed.

**Comment by walkah** [comment]
> Hm. We need a way to run kubo during tests.

Might make more sense to just mock for CI? 

Either way, can we get this merged so it's easier to build & run and work on a kubo testing strategy in a different PR?

**Comment by matheus23** [comment]
Yeah, let's merge.

**Pull request merged**